templates/pages/account/edit-payment-method.html

```diff
@@ -12,43 +12,61 @@ <h2 class="page-heading">{{lang 'forms.payment_methods.edit.heading'}}</h2>
             {{> components/common/alert-error customer.payment_methods.selected_payment_method.forms.error}}
         {{/if}}
 
-        <form action="{{customer.payment_methods.selected_payment_method.forms.action}}" data-address-form class="form" method="post">
-            <input type="hidden" name="bigpay_token" value="{{customer.payment_methods.selected_payment_method.bigpay_token}}">
-            <input type="hidden" name="currency_code" value="{{currency_selector.active_currency_code}}">
-
-            <h3 class="paymentMethodForm-heading">{{lang 'account.payment_methods.payment_method'}}</h3>
-
-            <div class="paymentMethodForm">
-                <dl class="paymentMethodForm-details">
-                    <dt class="paymentMethodForm-details-term">{{lang 'account.payment_methods.credit_card_number'}}</dt>
-                    <dd class="paymentMethodForm-details-description">**** **** **** {{customer.payment_methods.selected_payment_method.last_4}}</dd>
-                </dl>
-                <dl class="paymentMethodForm-details">
-                    <dt class="paymentMethodForm-details-term">{{lang 'account.payment_methods.expiration'}}</dt>
-                    <dd class="paymentMethodForm-details-description">{{customer.payment_methods.selected_payment_method.expiry_month}}/{{customer.payment_methods.selected_payment_method.expiry_year}}</dd>
-                </dl>
-                <div class="form-field">
-                    <input type="checkbox" value="default_instrument" name="default_instrument" id="default_instrument" data-label="{{lang 'forms.payment_methods.default_instrument'}}" class="form-checkbox" {{#if customer.payment_methods.selected_payment_method.default_instrument}}checked{{/if}}>
-                    <label class="form-label" for="default_instrument">{{lang 'forms.payment_methods.default_instrument'}}</label>
-                </div>
-            </div>
+        {{#with customer.payment_methods.selected_payment_method}}
```

Comment:
Yeah, this is not right. Should https://github.com/bigcommerce/cornerstone/pull/1603 takes precedence?